CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ Contributors:
 ### Fixes
 
 - Handle quoted values within test configs, such as `where` ([#3458](https://github.com/fishtown-analytics/dbt/issues/3458), [#3459](https://github.com/fishtown-analytics/dbt/pull/3459))
+- Fix type coercion issues when fetching query result sets ([#2984](https://github.com/fishtown-analytics/dbt/issues/2984), [#3499](https://github.com/fishtown-analytics/dbt/pull/3499))
```

Comment:
Is 0.20.0 the right home for this? Or should I slot it into 0.21.0?

Reply:
I'd prefer 0.21, rather than sneaking it in during RC time. Even though it is technically a bug fix, I imagine it _might_ be breaking behavior for folks who do a lot of custom query-results munging with Jinja + agate.